sources/packages/web/src/components/common/ConfirmEnrolment.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <check-permission-role :role="Role.StudentConfirmEnrolment">
+    <template #="{ notAllowed }">
+      <v-btn
+        v-if="isConfirmCOEEnabled"
+        variant="text"
+        color="primary"
+        @click="submitConfirmEnrolment"
+        :disabled="notAllowed"
```

Comment:
if an AEST does not have `StudentConfirmEnrolment` role, what do they see `Confirm Enrolment` btn/link, or its hidden for them?

Reply:
I didn't get the question.